kubernetes/celestia-local/scripts/init-bridge.sh

```diff
@@ -4,9 +4,7 @@ set -o errexit -o nounset
 
 ./celestia bridge init \
   --node.store "$home_dir/bridge" \
-  --core.ip 127.0.0.1
+  --core.ip 127.0.0.1\
```

Comment:
I think these flags are for the `start` command. Or can they also be used here and they determine what's set in `config.toml`?

Reply:
My understanding is they can be put here to configure the `config.toml`, or overridden in the start command